packages-exp/auth-exp/src/api/authentication/token.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { querystring } from '@firebase/util';
+
+import { performFetchWithErrorHandling } from '../';
+import { Auth } from '../../model/auth';
+
+export const _ENDPOINT = 'https://securetoken.googleapis.com/v1/token';
```

Comment:
can we pass this in on the auth object like we do with API_DOMAIN so that emulator can spoof it?

Reply:
Done